src/main/scala/com/horizon/exchangeapi/NodesRoutes.scala

```diff
@@ -802,7 +825,8 @@ trait NodesRoutes extends JacksonSupport with AuthenticationSupport {
               AuthCache.putNodeAndOwner(compositeId, Password.hash(reqBody.token), reqBody.token, owner)
               //AuthCache.ids.putNode(id, hashedTok, node.token)
               //AuthCache.nodesOwner.putOne(id, owner)
-              (HttpCode.PUT_OK, ApiResponse(ApiRespType.OK, ExchMsg.translate("node.added.or.updated")))
+              if (fivePercentWarning) (HttpCode.PUT_OK, ApiResponse(ApiRespType.OK, ExchMsg.translate("num.nodes.near.org.limit")))
```

Comment:
I did not, let me message them. They might want a different response code instead.

Reply:
Both of them were fine with this line its the 403 on the org limit of maxNodes that UI might have a problem with. They don't surface the error when a user hits the global maxNodes, it just errors. But for this one they need to surface it since you can change the org limit on maxNodes.